generators/client/templates/react/package.json.ejs

```diff
@@ -164,6 +164,7 @@ limitations under the License.
     <%_ if (protractorTests) { _%>
     "webdriver-manager": "12.1.5",
     <%_ } _%>
+    "@openapitools/openapi-generator-cli": "0.0.14-4.0.2",
```

Comment:
@ecostanzi,
It doesn't look appropriate to include generator sub-module dependency in the generated client application. Can't it be encapsulated in the module?

Reply:
Hi Vishal, what do you mean by saying "encapsulated in the module"? 

The documentations suggests:
> It is recommended to install the package as development dependency, because normally you only need this dependency during the development process

https://www.npmjs.com/package/@openapitools/openapi-generator-cli#package-mode

Reply:
Yes for me there is no problem to add this as dev dependency. Yes it does add significantly to what you have to download. But as it is useful, I don't see the problem of it being downloaded from npm rather than maven...

Reply:
IMO, it should be added as `generator-jhipster` dependency (not under dev) and not in the client applications dev dependency. Can you check if that works?

Reply:
Ok I'll let you know

Reply:
@vishal423 I opened issue https://github.com/jhipster/generator-jhipster/issues/10510 to discuss some changes. In that case I think that the dependency should remain in the package.json of the generated project.

Reply:
@ecostanzi, I will leave it to other members to decide. For me, I am planning to create a separate module that wouldn't contain it.